app/src/components/QuestMap.vue

```diff
@@ -130,12 +110,17 @@ function getOption(quests, progress) {
     completed: progress[node._id]?.completed || false,
     unlocked: node.requirements.every((req) => progress[req]?.completed),
   }))
-
-  console.log(nodes)
-
+  const lineAppearence = (node) => {
+    const { unlocked } = node
```

Comment:
Consider adding error handling when node is not found (should not happen but for safety)